testing/test_aux.out

```diff
@@ -47,16 +47,31 @@ RESULT skip security.passwords.http Port 80 is not open on target device.
 RESULT skip security.passwords.https Port 443 is not open on target device.
 RESULT skip security.passwords.telnet Port 23 is not open on target device.
 RESULT skip security.passwords.ssh Port 22 is not open on target device.
-RESULT fail security.passwords.http Default passwords have not been changed.
-RESULT fail security.passwords.https Default passwords have not been changed.
-RESULT fail security.passwords.telnet Default passwords have not been changed.
-RESULT fail security.passwords.ssh Default passwords have not been changed.
+RESULT skip security.passwords.http Port 80 is not open on target device.
+RESULT skip security.passwords.https Port 443 is not open on target device.
+RESULT skip security.passwords.telnet Port 23 is not open on target device.
+RESULT skip security.passwords.ssh Port 22 is not open on target device.
```

Comment:
Why this change?  What happened to testing the fail condition?

Reply:
I'll address this issue with the next commits - sorry, I completely missed this!